### DISCUSSION TITLE
Meilisearch Releases

### DISCUSSION THREAD

**Discussion by Lavendar77** [Homebrew/discussions#2202] [Getting started] [answered]
Installing version releases for `meilisearch` is not possible. 

I am currently working on a project using Laravel Scout and there seems to be an incompatibility issue with the latest version of MeiliSearch. The suggested solution was to use the meilisearch@v0.20.0, but there seems to be an issue with installing the specified version with homebrew: `brew install meilisearch@0.20.0`. 

Does anyone have a solution to this or same issue?

**Comment by SMillerDev** [comment, marked as answer]
Brew doesn't work like composer in that way. You can only really install the latest version from brew by default, unless the older version meets [strict requirements](https://docs.brew.sh/Versions). You can use [`brew extract`](https://docs.brew.sh/Manpage#extract---version---force-formula-tap) to maintain an older version for yourself though, although you will have to [maintain a tap](https://docs.brew.sh/How-to-Create-and-Maintain-a-Tap) for that.

**Comment by colivinggit** [comment]
We can install many older versions of other packages, but not Meilisearch.
Why? It's really annoying as Meilisearch is breaking on each new release.

**Reply by SMillerDev** [reply]
Because melisearch doesn't patch their old releases. If you don't like them breaking things, tell them. Brew has very little influence here.

**Comment by colivinggit** [comment]
Okay. Installing Docker then. Seems to be the only option to use older
versions.
